apps/client/src/index.css

```
@@ -3,11 +3,30 @@
+  --ink: #101828;
+  --ink-muted: #475467;
+  --ink-subtle: #98a2b3;
+  --rule: #e4e7ec;
+  --surface: #ffffff;
+  --surface-recessed: #fafafa;
+  --status-success: #079455;
+  --status-warning: #dc6803;
+  --status-danger: #d92d20;
 }
 
 @theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)